app.py

```diff
@@ -53,7 +53,9 @@ def cli(
     gh = Github()
 
     orgs = organizations.split(',')
-    repos = [repo.full_name for repo in gh.get_organization(org).get_repos() for org in orgs]
+
```

Comment:
please add some error/exception handling, and filter out archived repos too...

Reply:
done!